js/userInfo.js

```diff
@@ -25,5 +25,10 @@ $(document).ready(function () {
         }
     });
 
+
+
+
+    $(".dropdown-trigger").dropdown();
```

Comment:
You can't put initializers anywhere you want Lindsey ; ). Could you please add a init.js or materializeInit.js file and place it above the list of js files?

Reply:
Or token.js could be where the initializer goes as well